Common/Src/can_driver.c

```diff
@@ -0,0 +1,84 @@
+#include "can_driver.h"
+
+uint32_t tx_mailbox;
```

Comment:
Do you want this to just be visible in this .c file, or within other .c files as well? 

If it is the former, then maybe make it `static`. However, if it is the latter, maybe define it in the .h file, and use `extern` here and within any other source files it's used in.